### ISSUE TITLE
Increased serum sorbitol

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6102]
**Preferred term label:**
Increased serum sorbitol

**Synonyms**

**Definition (free text, please give PubMed ID)**
An increased level of sorbitol in the blood circulation.
PMID: 32367058

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0011013

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:618912 
PMID: 32367058  n=10/10

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as HP_0033124
